.github/workflows/tests.yml

```diff
@@ -114,6 +114,11 @@ jobs:
     name: Tiobe TICS
     runs-on: ubuntu-22.04
     needs: code-tests
+    env:
```

Comment:
Let's put them under [`env:`](https://github.com/canonical/microcluster/pull/292/files#diff-1db27d93186e46d3b441ece35801b244db8ee144ff1405ca27a163bfe878957fR9) at the root then the vars don't need to be duplicated and are present for all of the jobs.